src/coreComponents/mesh/simpleGeometricObjects/BoundedPlane.cpp

```diff
@@ -61,6 +61,32 @@ BoundedPlane::BoundedPlane( const string & name, Group * const parent ):
   m_points.resize( 4, 3 );
 }
 
+//constructor given two points, used for 2.5D problems
+BoundedPlane::BoundedPlane( const real64 oldX, const real64 oldY,
+                            const real64 newX, const real64 newY,
+                            const string & name, Group * const parent ):
+  SimpleGeometricObjectBase( name, parent ),
+  m_origin{ 0.0, 0.0, 0.0 },
+  m_normal{ 0.0, 0.0, 1.0 },
+  m_lengthVector{ 0.0, 0.0, 0.0 },
+  m_widthVector{ 0.0, 0.0, 0.0 },
+  m_tolerance()
+{
+  m_origin = { (oldX + newX)/2.0, (oldY + newY)/2.0, 0.0};
+  m_normal = { -(newY-oldY), newX-oldX, 0.0 };
+  m_lengthVector = { newX-oldX, newY-oldY, 0.0 };
+  m_widthVector = { 0.0, 0.0, 1.0 };
+  real64 norm = std::sqrt( pow( newX-oldX, 2 )+pow( newY-oldY, 2 ));
+  m_dimensions.resize( 2 );
+  m_dimensions[0] = norm+1e-4;
```

Comment:
Can you add some comments and some TODOs? These ad hoc tolerances seem a bit hacky. I think it's fine at this stage but it would be good to clean things up at some point.